xml/obs_build_process.xml

```diff
@@ -9,9 +9,10 @@
  xmlns:xlink="http://www.w3.org/1999/xlink" >
  <title>Build Process</title>
  <para>
-  Each package build is created in a fresh environment. This is done to
-  ensure that all dependencies are available and that every later build
-  produces identical results.
+  The build process is the actual process to create out of sources, binaries and
+  config new binaries. This process may run on the &obsa; server side or on a
```

Comment:
Improve word order/reduce sentence length: "The build process creates new binaries from sources, binaries, and config."
(I would expect something that is called "... process" to be "an actual process" in any case. :) )